src/Illuminate/Foundation/Auth/RegistersUsers.php

```diff
@@ -28,7 +30,11 @@ public function showRegistrationForm()
      */
     public function register(Request $request)
     {
-        $this->validator($request->all())->validate();
+        /** @var Validator $validator */
```

Comment:
while this may be valid docblocks, it's not consistent with the style of the rest of the FW